wordpress-mobile/WordPress-Android
# Switch to the new site creation flow and remove the old one

Fixes #9045 

**Note: Target branch is `issue/8511...`. We need to change it to `feature/master-site-creation` before we merge this PR.**

This PR removes all code related to the old site creation flow including Tracks events, resources etc.

We discussed it's better to remove the old(unused) code and we can use git to revert this PR if we need to fallback to the old site creation flow. I think one downside of this approach is that we'll throw away all the translations to other languages (if I'm not mistaken). Should we keep the string resources? Wdyt?

To test:
- Ideally make sure the site creation flow works as before. 

Update release notes:
- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
